cfgov/regulations3k/models/pages.py

```diff
@@ -567,7 +567,7 @@ def get_secondary_nav_items(request, current_page, sections=[], date_str=None):
             'title': section.title,
             'url': get_section_url(
                 current_page, section, date_str=date_str
-            ).lower(),
```

Comment:
Don't we still need this for the secondary nav items (in the left sidebar)?

Reply:
no, as `get_section_url` now does the `lower()`

Reply:
Ah, and `get_section_url` is also called in generating the next/previous links?

Reply:
correct